ouroboros-consensus/src/Ouroboros/Consensus/Node/ProtocolInfo/Byron.hs

```diff
@@ -79,7 +120,7 @@ protocolInfoByron (NumCoreNodes numCoreNodes) nid params gc mLeader =
               , pbftGenesisConfig   = gc
               , pbftGenesisHash     = coerce Cardano.Genesis.configGenesisHeaderHash gc
               , pbftEpochSlots      = Cardano.Genesis.configEpochSlots gc
-              , pbftGenesisDlg      = Cardano.Genesis.configHeavyDelegation gc
+              , pbftGenesisDlg      = delegation
```

Comment:
I wouldn't bother changing this, it's clearer how it was. Achieving sharing here doesn't buy anything.

Reply:
Done.